torch/_inductor/graph.py

```diff
@@ -1238,7 +1240,12 @@ def debug(msg: str) -> None:
             if (
                 n.op == "call_function"
                 and n.target is not operator.getitem
-                and fallback_node_due_to_unsupported_type(n)
+                and (
+                    fallback_node_due_to_unsupported_type(n)
+                    or BisectionManager.disable_subsystem(
+                        "inductor", "lowerings", lambda: repr(n)
```

Comment:
This is an interesting thing to do, because while I can see why it is useful, I could imagine it also could trigger latent bugs, where the lowering path is fine but the fallback path is buggy.